File: trunk/net.tenteCsharp.templatesProject/src_code/lightManagement/IGeneralLightNotify.java
package lightManagement;
import lightManagement.LightManagement.*;

public interface IGeneralLightNotify{
	
	public void switchOffAllLights(String floorId,String roomId);	
	public void switchOnAllLights(String floorId,String roomId);	
	public void changeAllLightsIntensity(int intensity,String floorId,String roomId);

}

File: trunk/pseudoCodeGeneratorElio/src-gen/lightManagement/IGeneralLightNotify.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	public interface IGeneralLightNotify
	{
		void switchOffAllLights(String floorId, String roomId);
		void switchOnAllLights(String floorId, String roomId);
		void changeAllLightsIntensity(int intensity, String floorId, String roomId);

	}
}
